src/EA.Weee.Domain/Organisation/Organisation.cs

```diff
@@ -53,7 +53,7 @@ private set
             {
                 if (value != null && (value.Length < 7 || value.Length > 15))
                 {
-                    throw new InvalidOperationException("Company registration number must be 7 to 15 characters long");
+                    throw new InvalidOperationException("company registration number must be 7 to 15 characters long");
```

Comment:
Think this should be capitalised, as its start of sentence.

Reply:
restored